packages/governance/test/swingsetTests/contractGovernor/governor.js

```diff
@@ -0,0 +1,41 @@
+// @ts-check
+
+import { Far } from '@agoric/marshal';
+import { E } from '@agoric/eventual-send';
+import { setupGovernance } from './governParam';
+
+/** @type {ContractStartFn} */
+const start = async zcf => {
+  const zoe = zcf.getZoeService();
+
+  const startGovernedInstance = async (
+    registrar,
+    governedContractInstallation,
+    issuerKeywordRecord,
+    customTerms,
+  ) => {
+    const { creatorFacet, instance: governedInstance } = await E(
+      zoe,
+    ).startInstance(
+      governedContractInstallation,
+      issuerKeywordRecord,
+      customTerms,
+    );
+
+    /** @type {Promise<ParamManagerFull>} */
+    const paramManager = E(creatorFacet).getParamManager();
+
+    // connect the vote outcome to the paramManager
+    const setParamByVoting = setupGovernance(paramManager, registrar);
```

Comment:
We pass the paramManager to an imported function that allows us to set a parameter through voting